articles/quickstart/spa/auth0-react/02-calling-an-api.md

```diff
@@ -0,0 +1,142 @@
+---
+title: Call an API
+description: This tutorial demonstrates how to make API calls for protected resources on your server.
+budicon: 448
+topics:
+  - quickstarts
+  - spa
+  - react
+  - login
+github:
+  path: Sample-01
+contentType: tutorial
+useCase: quickstart
+---
+<!-- markdownlint-disable MD002 MD034 MD041 -->
+
+Your React application may need to access protected data from an API that you are also [securing with Auth0](https://auth0.com/docs/microsites/protect-api/protect-api). For that use case, you'll need to retrieve an access token to authorize any calls you make to that secured API.
+
+For this example, you'll use the [Auth0 Management API](https://auth0.com/docs/api/management/v2) to quickly test how to make calls to a protected external API.
```

Comment:
we should point out that this could be your own API. We are using the Auth0 Management API simply because it is already available and create a self-contained tutorial.